lib/tools/apks-utils.js

```diff
@@ -275,6 +275,32 @@ apksUtilsMethods.isTestPackageOnlyError = function isTestPackageOnlyError (outpu
   return /\[INSTALL_FAILED_TEST_ONLY\]/.test(output);
 };
 
+
+/**
+ * Returns true if the given output had INSTALL_PARSE_FAILED_NO_CERTIFICATES word.
+ * Usually the apk does not have proper signature scheme. It needs to be signed properly,
+ * or the apk needs to be updated.
+ * https://developer.android.com/tools/apksigner#options-sign
+ *
+ * @param {string} output - The adb command output
+ * @returns {boolean} If the output includes INSTALL_PARSE_FAILED_NO_CERTIFICATES
+ */
+apksUtilsMethods.isInsufficientSignatureScheme = function isInsufficientSignatureScheme (output) {
```

Comment:
I am not quote convinced we need to mixin these helpers directly into ADB instance. I would rather keep themon the private level